staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -110,6 +138,10 @@ type byPriority struct {
 func (a byPriority) Len() int      { return len(a.specs) }
 func (a byPriority) Swap(i, j int) { a.specs[i], a.specs[j] = a.specs[j], a.specs[i] }
 func (a byPriority) Less(i, j int) bool {
+	// All local specs will come first
+	if a.specs[i].apiService.Spec.Service == nil {
```

Comment:
I completely agree. What I tried to say is: this is outside of the scope of this PR (The problem was not introduced here as far as I can tell. The line added here is merely simulate what we were doing before this PR that aggregating `kubeApiServer` as whole with aggregator server's spec then proceed to any non-local spec). We should definitely find a solution for this and fix it.